e2e-tests/cypress/e2e/pages/insurance/declarations/confidentiality.js

```diff
@@ -0,0 +1,35 @@
+import { FIELD_IDS } from '../../../../../constants';
+
+const {
+  INSURANCE: {
+    DECLARATIONS: { AGREE_CONFIDENTIALITY },
+  },
+} = FIELD_IDS;
+
+const confidentialityPage = {
+  listItems: {
+    intro: () => cy.get("[data-cy='paragraph-1']"),
+    level1: {
+      item1: () => cy.get("[data-cy='list-level-1-item-1']"),
```

Comment:
could we simplify this to be `item: () => cy.get("[data-cy='list-level-${level}-item-${item}']"),`

Reply:
I did think about this, I think it would end up too vague/ambiguous as we'd end up with tests like this:

```js
cy.checkText(item(1, 2), content[1].text);
cy.checkText(item(3, 1), content[2].text);
cy.checkText(item(2, 4), content[3].text);
```

Extracting the numbers would add a lot of lines of code as well.

What do you think?